src/sentry/features/handler.py

```diff
@@ -33,9 +35,10 @@ def has_for_batch(self, batch: "FeatureCheckBatch") -> Mapping["Project", bool]:
     def batch_has(
         self,
         feature_names: Sequence[str],
-        actor: "User",
-        projects: Optional[Sequence["Project"]] = None,
-        organization: Optional["Organization"] = None,
+        actor: User,
+        projects: Optional[Sequence[Project]] = None,
+        organization: Optional[Organization] = None,
+        user: Optional[Organization] = None,
```

Comment:
Little confused by this variable naming, where it is `user` but expecting an Organization?

Reply:
@wmak oops thats a typo